src/components/BoxColor.js

```diff
@@ -0,0 +1,13 @@
+function BoxColor({ r, g, b, hex }) {
```

Comment:
`hex` was not specified as a prop value; if you wanted to display it, you should have calculated it from the rgb value